sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/MethodsShouldUseBaseTypes.cs

```diff
@@ -729,6 +733,26 @@ protected void MethodTwo(Something.Foo f)
         {
             var x = f.IsFoo;
         }
+

```

Comment:
[optional] perhaps worth splitting the tests with Foo and Bar arguments to separate classes - in the same namespace?

Reply:
Well they are both linked to accessibility so I will keep them as it.